locales/en.default.schema.json

```
@@ -114,49 +114,29 @@
     "colors": {
       "name": "Colors",
       "settings": {
-        "colors_solid_button_labels": {
-          "label": "Solid button label",
-          "info": "Used as foreground color on accent colors."
+        "background": {
+          "label": "Background"
         },
-        "colors_accent_1": {
-          "label": "Accent 1",
-          "info": "Used for solid button background."
+        "background_gradient": {
+          "label": "Background gradient",
+          "info": "Background gradient replaces background where possible."
         },
-        "gradient_accent_1": {
-          "label": "Accent 1 gradient"
-        },
-        "colors_accent_2": {
-          "label": "Accent 2"
-        },
-        "gradient_accent_2": {
-          "label": "Accent 2 gradient"
-        },
-        "header__1": {
-          "content": "Primary colors"
-        },
-        "header__2": {
-          "content": "Secondary colors"
-        },
-        "colors_text": {
-          "label": "Text",
-          "info": "Used as foreground color on background colors."
+        "text": {
+          "label": "Text"
         },
-        "colors_outline_button_labels": {
-          "label": "Outline button",
-          "info": "Also used for text links."
+        "button_background": {
+          "label": "Solid button background"
         },
-        "colors_background_1": {
-          "label": "Background 1"
+        "button_label": {
+          "label": "Solid button label"
         },
-        "gradient_background_1": {
-          "label": "Background 1 gradient"
+        "secondary_button_label": {
+          "label": "Outline button"
         },
-        "colors_background_2": {
-          "label": "Background 2"
+        "shadow": {
+          "label": "Shadow"
         },
-        "gradient_background_2": {
-          "label": "Background 2 gradient"
-        }
+        "info": "Scheme 1 is used as a fallback for sections and templates without a color scheme setting."
```

```suggestion
        "info": "Scheme 1 is used for areas without a color scheme setting."
```

@silviacela @katycobb Can you please help remind me if this was the last version we aligned on? 🙏